src/screens/SearchScreen.js

```diff
@@ -3,49 +3,40 @@ import {View, Text, ActivityIndicator, StyleSheet, Platform, Keyboard} from 'rea
 import {Button} from 'react-native-elements';
 import {MapView} from 'expo';
 import {Icon} from 'react-native-elements';
+import {connect} from 'react-redux';
+
+import {initializeMap, updateViewStyle, mapLoaded, updateRegion} from "../actions";
```

Comment:
Refactored to use `connect` helper and corresponding actions to get away from `setState`.